control-plane/api-gateway/controllers/gateway_controller.go

```diff
@@ -363,10 +363,6 @@ func SetupGatewayControllerWithManager(ctx context.Context, mgr ctrl.Manager, co
 			source.NewKindWithCache(&corev1.Secret{}, mgr.GetCache()),
 			handler.EnqueueRequestsFromMapFunc(r.transformSecret(ctx)),
 		).
-		Watches(
-			source.NewKindWithCache(&gwv1beta1.ReferenceGrant{}, mgr.GetCache()),
-			handler.EnqueueRequestsFromMapFunc(r.transformReferenceGrant(ctx)),
-		).
```

Comment:
this was duplicated a few lines above, so just dropped it while I was adding this.